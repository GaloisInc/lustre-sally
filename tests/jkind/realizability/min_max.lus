-- Unrealizable

node main(max, min, init_max : int) returns (ok : bool);
let
  assert init_max > 20;
  ok = max = (init_max -> 1 + pre max) and
       min = (0 -> 2 + pre min) and
       max > min;
  --%PROPERTY ok;
  --%REALIZABLE init_max;
tel;
