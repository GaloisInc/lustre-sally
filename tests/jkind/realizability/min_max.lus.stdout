[Error] Parse error at 10:17

